Add Node promotion zone label support (#201)
harvester/docs: docs/install/pxe-boot-install.md

For more information about Harvester configuration, please refer to the [Harvester configuration](./harvester-configuration.md) page.

> The first node always defaults to be a management node of the cluster. When there are three or more nodes, the two other nodes that first joined are automatically promoted to management nodes to form a HA cluster.

Reviewer: ```suggestion
By default, the first node will become the management node of the cluster. When there are three or more nodes, the two other nodes that first joined are automatically promoted to management nodes to form a HA cluster.
```

> If you want to make sure that only compute nodes in zones different from the existing management nodes can be promoted, you can specify the node label `topology.kubernetes.io/zone` in the [os.labels](./harvester-configuration.md#oslabels) of the first node and other compute nodes that you want to promote. In this case, at least three different zones are required.

Reviewer 2: ```suggestion
If you want to promote management nodes from different zones, you can add the node label `topology.kubernetes.io/zone` in the [os.labels](./harvester-configuration.md#oslabels) config. In this case, at least three different zones are required.
```

> If you want to make sure that the node promotion mechanism will only promote compute nodes in zones different from existing management nodes to management, you can specify node label `topology.kubernetes.io/zone` in the [os.labels](./harvester-configuration.md#oslabels) of first node and other compute nodes that you want to be promoted to management node. In this case, at least three different zones are required.

Reviewer: ```suggestion
If you want to make sure that only compute nodes in zones different from the existing management nodes can be promoted, you can specify the node label `topology.kubernetes.io/zone` in the [os.labels](./harvester-configuration.md#oslabels) of the first node and other compute nodes that you want to promote. In this case, at least three different zones are required.
```
